vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -54,12 +54,7 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
           ItemCache.prototype.doEnsureSubCacheForScaledIndex = tryCatchWrapper(function (scaledIndex) {
```

Comment:
This whole method override is now unnecessary since it 100% repeats the [original method](https://github.com/vaadin/web-components/blob/f5ff5170fbf8453095e6a711239dc7e6881699e0/packages/grid/src/vaadin-grid-data-provider-mixin.js#L74-L80) in the web component.

Reply:
I knew there was something I wanted to check... Perfect, thanks for the heads up. Removed.

Looks like Github is having issues with showing the commit though.

Reply:
There are currently methods that invoke `doEnsureSubCacheForScaledIndex `. They need to be updated to invoke `ensureSubCacheForScaledIndexOriginal` instead.